mlflow/tracing/trace_manager.py

```diff
@@ -26,8 +26,13 @@ class _Trace:
     span_dict: Dict[str, MlflowSpanWrapper] = field(default_factory=dict)
 
     def to_mlflow_trace(self) -> Trace:
-        span_list = [span.to_mlflow_span() for span in self.span_dict.values()]
-        return Trace(self.trace_info, TraceData(spans=span_list))
+        trace_data = TraceData()
+        for span in self.span_dict.values():
+            trace_data.spans.append(span.to_mlflow_span())
+            if span.parent_span_id is None:
```

Comment:
Is it ever possible to have multiple spans whose parent is None? How should we handle that case? Pick the first span with parent None?

Reply:
Theoretically not, because we don't allow calling `MlflowClient.start_trace()` inside a span, and in fluent API it is guaranteed. In case if it happens, that means the trace is completely broken, so taking the first or last (current approach) doesn't  matter I guess?